aea/decision_maker/base.py

```diff
@@ -301,11 +306,14 @@ def _handle_tx_message(self, tx_message: TransactionMessage) -> None:
         :param tx_message: the transaction message
         :return: None
         """
-        score_diff = self.preferences.get_score_diff_from_transaction(self.ownership_state, tx_message)
-        if score_diff >= 0.0:
-            pass  # TODO: implement tx flow
-            # envelope = Envelope()
-            # self.outbox.put(envelope)
+        m_address = generate_address_from_public_key(self._wallet.public_keys['fetchai'])
+        api = LedgerApi("127.0.0.1", 8000)
+        if tx_message.get("amount") <= api.tokens.balance(m_address):
+            m_entity = Entity.from_hex(self._wallet.crypto_objects['fetchai'].private_key)
+            to = generate_address_from_public_key(tx_message.get("counterparty"))
+            api.sync(api.tokens.transfer(m_entity, to, 1, 1))
```

Comment:
what do 1, 1 stand for? please create constants :)

Reply:
Nah, I just forgot to change them back to tx_message.get("amount") and tx_message.get("sender_tx_fee"). These were changes I made before we fix the ledger.